java-util/src/test/java/io/druid/java/util/http/client/JankyServersTest.java

```diff
@@ -309,16 +314,10 @@ public void testHttpEchoServer() throws Throwable
               new StatusResponseHandler(StandardCharsets.UTF_8)
           );
 
-      Throwable e = null;
-      try {
-        response.get();
-      }
-      catch (ExecutionException e1) {
-        e = e1.getCause();
-      }
+      expectedException.expect(ExecutionException.class);
```

Comment:
Would you please improve `testHttpsEchoServer` as well using expectedException?

Reply:
Done.